src/sentry/grouptype/grouptype.py

```diff
@@ -12,7 +11,7 @@ class GroupType:
     slug: str
     description: str
     category: int
-    ignore_limit: int = DEFAULT_GROUPHASH_IGNORE_LIMIT
+    ignore_limit: int = 3  # CEO temp fix - this is the value of DEFAULT_GROUPHASH_IGNORE_LIMIT
```

Comment:
This is due to a circular import issue that's resolved in the larger PR this is related to - unless I make this more complicated than I want to, I'll leave this for now.